### DISCUSSION TITLE
traefik dynamic/static config - where?

### DISCUSSION THREAD

**Discussion by 1mfaasj** [tteck/Proxmox#3751] [Help Needed] [answered]
maybe a stupid question for a new user to your scripts, but I just rolled out the Traefik LXC and am looking for the static and dynamic config (file) that Traefik needs. Would you like to help me on the right track?
I looked in the Console but can't find anything that starts with "traefik". There is also no traefik folder in the 'etc' folder.
Thanks!

**Comment by tteck** [comment, marked as answer]
Have you looked through the code?
https://github.com/tteck/Proxmox/blob/main/install/traefik-install.sh
